### PR TITLE
feat: batch operator id conversions

### PR DESCRIPTION
**Pull request by 0x0aa0**
Adds two view functions to OperatorStateRetriever:
- getBatchOperatorId: batch conversion of operator address to operator id
- getBatchOperatorFromId: batch conversion of operator id to operator address

### CODE REVIEW

File: src/OperatorStateRetriever.sol [hunk 1]
```diff
@@ -178,4 +178,25 @@ contract OperatorStateRetriever {
         }
         return quorumBitmaps;
     }
+
+    function getBatchOperatorId(
```
Comment (jianoaix) [inline src/OperatorStateRetriever.sol:188]: These two methods will revert if any of the ID/address doesn't exist?

**Review by jianoaix — COMMENTED** [review]
LG

File: src/OperatorStateRetriever.sol (hunk 1 above)
Comment (ChaoticWalrus) [reply src/OperatorStateRetriever.sol:188]: as written, these methods currently will _not_ revert, but instead return `bytes32(0)` as the ID of any operator who has never registered, or `address(0)` for any operatorID that has never been used.

relevant code:
https://github.com/Layr-Labs/eigenlayer-middleware/blob/feat/batch-operatorid-view/src/BLSApkRegistry.sol#L275-L276
https://github.com/Layr-Labs/eigenlayer-middleware/blob/feat/batch-operatorid-view/src/RegistryCoordinator.sol#L796-L804

@jianoaix what would your preferred behavior here be?

**Review by ChaoticWalrus — CHANGES_REQUESTED** [review]
Looks fine. Would be good to resolve the thread around reversions + add some documentation, prior to merging.

File: src/OperatorStateRetriever.sol (hunk 1 above)
Comment (jianoaix) [reply src/OperatorStateRetriever.sol:188]: Yep, that's more desirable! Could we put this in the comment of the function?

Comment (0x0aa0) [reply src/OperatorStateRetriever.sol:182]: No they will just return a 0 value at that index

**Review by jianoaix — COMMENTED** [review]
LGTM

**Review by ChaoticWalrus — APPROVED** [review]
LGTM.
FYI, the 2 automated test failures are unreleated to this PR and are getting fixed in a separate PR.

**Pull request merged**